test/over_react_redux/connect_flux_test.dart

```diff
@@ -0,0 +1,776 @@
+// Copyright 2020 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:over_react/over_react.dart';
+import 'package:over_react/over_react_flux.dart';
+import 'package:over_react/over_react_redux.dart';
+import 'package:test/test.dart';
+
+import '../test_util/test_util.dart';
+import 'fixtures/connect_flux_counter.dart';
+import 'fixtures/connect_flux_store.dart';
+import 'fixtures/counter.dart';
+import 'fixtures/redux_actions.dart';
+import 'fixtures/store.dart' as redux_store;
+
+// ignore_for_file: avoid_types_on_closure_parameters
+
+main() {
+  group('connectFlux', () {
+    UiFactory<ConnectFluxCounterProps> ConnectedCounter;
+    TestJacket<ConnectFluxCounterComponent> jacket;
+    final counterRef = createRef<ConnectFluxCounterComponent>();
+
+    JsConnectOptions connectOptions;
+    var originalConnect = mockableJsConnect;
+
+    mockableJsConnect = ([
```

Comment:
Can we do this inside a `setUp` or `setUpAll`, and then also tear it down in `tearDown`/`tearDownAll` by resetting it to `originalConnect`? That way we'll leave the test environment clean.